code/game/gamemodes/objective.dm

```diff
@@ -414,6 +417,10 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 	/// Overrides the hijack speed of any antagonist datum it is on ONLY, no other datums are impacted.
 	var/hijack_speed_override = 1
 
+/datum/objective/hijack/on_obj_given()
+	owner.murderbone_types |= MIND_MURDERBONE_GENERAL | MIND_MURDERBONE_HIJACK
+	message_admins("[key_name(owner)] has acquired murderbone pass.")
```

Comment:
Can't objectives have multipler owners, for example in the case of incursion?

Reply:
no longer used as that way